### PR TITLE
Fix #177441 celebfapper.com

### LINKED ISSUE #177441: celebfapper.com
### Issue URL (Anti Adblock Script)
[NSFW] [https://celebfapper.com/photo/1708387192/sydney-sweeney-fap-nude-naked-tits-boobs](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcelebfapper.com%2Fphoto%2F1708387192%2Fsydney-sweeney-fap-nude-naked-tits-boobs)

### [NSFW] Screenshots
  [Screenshot 1](https://cdn.adguard.info/sitereports/1kq2pui9fb34kkkwcoocosso48ocww0mf786jp8hpw.png?nc=1)

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.46
System version: | Windows 10
Browser: | Opera
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances

### PR DESCRIPTION
**Pull request by Numci**
Fix issue #177441

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 !
 ! NOTE: Specific rules
 !
+celebfapper.com##.row
```
Comment (BlazDT) [inline BaseFilter/sections/specific.txt:346]: Dangerous rule in generic. Also only seeing one element with some elements in code.

File: BaseFilter/sections/specific.txt [hunk 2]
```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
...
+/invoke.js$script,domain=celebfapper.com
```
Comment (BlazDT) [inline BaseFilter/sections/specific.txt:347]: Only useful for iOS due to missing regexp support ( existing rule ` /[0-9a-f]{32}\/invoke\.js/$script,third-party,redirect-rule=noopjs`)

**Review by BlazDT — CHANGES_REQUESTED** [review]
For adblock:
`@@||celebfapper.com/js/adManager.js`

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (Numci) [reply BaseFilter/sections/specific.txt:346]: Remove this rule, it is not needed. Base filter already block it

File: BaseFilter/sections/specific.txt (hunk 2 above)
Comment (Numci) [reply BaseFilter/sections/specific.txt:347]: Remove this too, base filter blocked it already too

**Review by BlazDT — CHANGES_REQUESTED** [review]
Please add unblock rule I posted before (iOS/android cb).

File: BaseFilter/sections/allowlist.txt [hunk 3]
```diff
@@ -108,6 +108,8 @@ $popup,third-party,domain=dood.ws,badfilter
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/177441
+@@||celebfapper.com/js/adManager.js
```
Comment (BlazDT) [inline BaseFilter/sections/allowlist.txt:112]: To antiadblock instead of allowlist please.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/allowlist.txt (hunk 3 above)
Comment (BlazDT) [reply BaseFilter/sections/allowlist.txt:112]: Near line 1093 (! NOTE: Regular rules)

**Review by BlazDT — APPROVED** [review]

File: BaseFilter/sections/antiadblock.txt [hunk 4]
```diff
@@ -1092,6 +1092,8 @@ pornhub.com,pornhub.org,pornhub.net#%#//scriptlet('prevent-setTimeout', '\x41\x6
 !
 ! NOTE: Regular rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/177441
+@@||celebfapper.com/js/adManager.js
```
Comment (Alex-302) [inline BaseFilter/sections/antiadblock.txt:1096]: Please add as less specific solution (not sure if they switch to another request, but scriptlets are good alternative)
`celebfapper.com/#%#//scriptlet('prevent-setTimeout', 'adblocktag')`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Sergey-Lyapin — APPROVED** [review]

**Review by BlazDT — CHANGES_REQUESTED** [review]
Remove / in celebfapper.com/ ( aka only the one after .com )

File: BaseFilter/sections/specific.txt [hunk 5]
```diff
@@ -343,6 +343,7 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 !
 ! NOTE: Specific rules
 !
+celebfapper.com/#%#//scriptlet('prevent-setTimeout', 'adblocktag')
```
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:346]: Error 
```suggestion
celebfapper.com#%#//scriptlet('prevent-setTimeout', 'adblocktag')
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/antiadblock.txt (hunk 4 above)
Comment (Sergey-Lyapin) [reply BaseFilter/sections/antiadblock.txt:1096]: And please move the scriptlet above the exception rule

**Review by BlazDT — CHANGES_REQUESTED** [review]
Rule looks good now. Please move to other rule in antiadblock.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**